src/app/design/frontend/base/default/template/payment/form/omise/omiseoffsiteinstallment.phtml

```diff
@@ -1,11 +1,19 @@
 <?php
 $code = $this->getMethodCode();
 $backends = $this->getInstallmentBackends();
+$isInterestFree = $this->isInterestFree();
+
+$quote = Mage::helper('checkout')->getQuote();
+$saleAmount = $quote->getBaseGrandTotal();
+$currencyHelper = Mage::app()->getLocale()->currency($quote->getBaseCurrencyCode());
+
 ?>
 
 <ul id="payment_form_<?php echo $code; ?>" class="banks-list form-list" style="display:none;">
 
-    <?php if (count($backends)) { foreach ($backends as $backend) { ?>
```

Comment:
I would strongly start using php designed for this purposes conditonal statements:
`if ($condition):`

`endif;`
`foreach`
`endforeach`

Reply:
Never used it, never liked it since I first saw it. I'd recommend not using it personally

Reply:
why do you recommend not to use it?

it is not good, exactly as not good is:
```
public function foo()
{
?>
    <div>
        This is text to print
    </div>
<?php
}
```

it is no mistake, but it is not good...

Reply:
It's just an [alternative syntax](https://php.net/manual/en/control-structures.alternative-syntax.php) for control structures . I think code should be consistent in the use of control structures - not mixing 2 different styles depending on situations. I prefer to stay all with one syntax. I think the alternative syntax is ugly